hasher-matcher-actioner/webapp/src/components/SubmitContentFields.tsx

```diff
@@ -27,53 +27,6 @@ type ContentFieldProps = {
   handleInputChange: (e: React.SyntheticEvent) => void;
 };
 
-export function ContentUniqueIdField({
```

Comment:
nit: Ideally we either have this file for all fields. Could you move the `<Form.Group>` for content type choice cards and 'Unique ID for Content' here?